app/os_macos.go

```diff
@@ -563,6 +574,16 @@ func gio_onFocus(view C.CFTypeRef, focus C.int) {
 	w.SetCursor(w.cursor)
 }
 
+//export gio_onResponder
+func gio_onResponder(view C.CFTypeRef, first C.int) {
```

Comment:
What about `gio_onFocus`, `windowDidBecomeKey`, `windowDidResignKey`? It seems to me they should be removed by this change. If not, why not?

Reply:
I think each one fits one purpose. The Responder seems to be related to the current window, and Key is global. In the end:

- If you minimize the window: the ResignKey is triggered, but resignFirstResponder is not.
- If you click in another child window: the ResignKey is not triggered, but the resignFirstResponder will be triggered.

Seems that in some conditions, both will notify.

Reply:
> I think each one fits one purpose. The Responder seems to be related to the current window, and Key is global. In the end:
> 
>     * If you minimize the window: the ResignKey is triggered, but resignFirstResponder is not.
> 
>     * If you click in another child window: the ResignKey is not triggered, but the resignFirstResponder will be triggered.
> 
> 
> Seems that in some conditions, both will notify.

So in total, Key is what we want for StageInactive, and Responder is what we want to use for focus, right?